Make fuse_operation struct globally accessible and comment-out
main() function.

+++ src/lib/exfat/fuse/main.c
@@ -296,7 +296,7 @@
 	exit(1);
 }
 
-static struct fuse_operations fuse_exfat_ops =
+struct fuse_operations fuse_exfat_ops =
 {
 	.getattr	= fuse_exfat_getattr,
 	.truncate	= fuse_exfat_truncate,
@@ -403,6 +403,7 @@
 	return options;
 }
 
+#if 0
 int main(int argc, char* argv[])
 {
 	struct fuse_args mount_args = FUSE_ARGS_INIT(0, NULL);
@@ -550,3 +551,4 @@
 	fuse_destroy(fh);
 	return 0;
 }
+#endif
